region/crateria/east/West Ocean.json

```diff
@@ -2676,6 +2676,19 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [13, 13],
+      "name": "Ripper2 Farm",
+      "requires": [
+        {"resetRoom": {
+          "nodes": [1, 4, 5],
+          "mustStayPut": false
+        }},
+        "ScrewAttack",
+        {"partialRefill": {"type": "Super", "limit": 4}},
+        {"partialRefill": {"type": "Energy", "limit": 80}}
```

Comment:
Probably not a problem, but i wonder if this should have some sort of knowledge check on it. Without it, the player could still farm the skulterras and get the energy, so it is mostly about the Supers.